src/ExpressionParser.php

```diff
@@ -531,11 +532,7 @@ public function parsePostfixExpression($node)
     public function getFunctionNode($name, $line)
     {
         if (null !== $alias = $this->parser->getImportedSymbol('function', $name)) {
-            $arguments = $this->createArguments($line);
-            $node = new MethodCallExpression($alias['node'], $alias['name'], $arguments, $line);
-            $node->setAttribute('safe', true);
```

Comment:
The `safe` attribute is always set to `true` in all usages, so I've moved it to the constructor instead.